spago.yaml

```
@@ -62,3 +63,106 @@ workspace:
       git: https://github.com/purescript/registry-dev.git
       ref: 6a803c37577af368caa221a2a06d6be2079d32da
       subdir: foreign
+    html-parser-halogen:
+      dependencies:
+        - halogen
+        - string-parsers
+      git: https://github.com/rnons/purescript-html-parser-halogen.git
+      ref: 035a51d02ba9f8b70c3ffd9fe31a3f5bed19941c
+    jest:
+      dependencies:
+        - aff
+        - aff-promise
+        - effect
+        - foldable-traversable
+        - prelude
+        - psci-support
+      git: https://github.com/klntsky/purescript-jest.git
+      ref: 7feaa5a880fc75002c4eca312993174e7220252b
```

Can move back to upstream here as well:
```suggestion
      git: https://github.com/nonbili/purescript-jest.git
      ref: caf2032f2e5828337e897a99f5359c00e91cb0ee
```